source/public_collections_advanced_search.md

```diff
@@ -0,0 +1,97 @@
+# Public Collections Advanced Search
+
+**_Note that the user-friendliness of advanced searching will improve in the
+coming months._**
+
+### The Default Search Behaviour
+
+When you enter a term into the search bar, the following fields are searched by
+default when no explicit field is given (combined with the `OR` operator):
+`name__icontains`, `owner__username__icontains`,
+`settings__description__icontains`, `summary__icontains`,
+`tags__name__icontains`, `uid__icontains`.
```

Comment:
I think this is great for power users (maybe the target audience of this article?), but I can imagine most people better understanding something less jargon-y like
> When you enter a term into the search bar without specifying a field, your query will return results where that term, regardless of capitalization, can be found in:
> * The name of the survey, collection, question, block, or template;
> * The owner's username;
> * The description;
> * The summary, which contains all question labels; [and maybe other stuff? I don't remember]
> * The name of any assigned tag;
> * The object's UID.